node/src/chain_spec.rs

```diff
@@ -41,6 +42,15 @@ pub fn authority_keys_from_seed(s: &str) -> (AuraId, GrandpaId) {
 
 pub fn development_config() -> Result<ChainSpec, String> {
 	let wasm_binary = WASM_BINARY.ok_or("Development wasm binary not available".to_string())?;
+	let properties = Some(
+		json!({
```

Comment:
This only seems to indicate how information about 1 token, how does one validate balances of any resource ID?